cmd/earth/main.go

```diff
@@ -171,7 +171,21 @@ func main() {
 		color.NoColor = true
 	}
 
-	app := newEarthApp(ctx, conslogging.Current(colorMode))
+	padding := conslogging.DefaultPadding
+	customPadding, ok := os.LookupEnv("TARGET_PADDING")
+	if ok {
+		targetPadding, err := strconv.Atoi(customPadding)
+		if err == nil {
+			padding = targetPadding
```

Comment:
Only assign when its not nil, and just silently use the default if the env var is not set correctly. Didn't seem worth logging a message about, would likely be apparent what happened?